css-backgrounds-4/Overview.bs

```diff
@@ -185,26 +185,32 @@ Painting Area: the 'background-clip' property</h3>
 
 	<pre class=propdef>
 	Name: border-top-color, border-right-color, border-bottom-color, border-left-color
-	Value: <<color>>#
+	Value: <<color>> || <<1d-image>>
 	Initial: currentcolor
 	Applies to: all elements
 	Inherited: no
 	Percentages: n/a
-	Computed Value: the computed color
+	Computed Value: the computed color and/or a one-dimensional image function
 	Animation type: see prose
 	</pre>
 
 	<pre class="propdef shorthand">
 	Name: border-color
-	Value: <<color>>#{1,4}
+	Value: [ <<color>> || <<1d-image>> ]{1,4}
 	</pre>
 
 	These properties set the foreground color of the border specified
 	by the 'border-style' properties.
-	If a list of values is provided, the border is split into equal width bands of each color
-	along the direction of the side the border is applied on
-	(i.e. split horizontally on left and right borders and vertically on top and bottom borders),
-	starting outwards.
+
+	The stripes defined by <<1d-image>> follow the direction of
+	the side the border is applied on (i.e. horizontally on left and
+	right borders and vertically on top and bottom borders), starting outwards.
+	With that, the corresponding 'border-*-width' property defines the total
+	width of the stripes.
+
+	The stripes are painted above a given <<color>>. This allows the <<color>>
+	value to serve as a fallback, so any transparent stripes let the color
+	shine through.
```

Comment:
What happens if no color is specified? Do we get transparent or currentColor or...?

Reply:
Good point! I'd say `currentcolor` as it's the initial value. I'll add that for now, though we may still discuss whether that's the best choice.

Sebastian

Reply:
Because we've now defined to have either a color or a 1D image that has transparent black as its default color, there's no need to define this anymore.

Sebastian